chpl/chpl-service/src/main/java/gov/healthit/chpl/surveillance/report/builder2019/ActivitiesAndOutcomesWorksheetBuilder2019.java

```diff
@@ -19,32 +19,37 @@ public ActivitiesAndOutcomesWorksheetBuilder2019(SurveillanceReportManager repor
         super(reportManager, detailsManager, survManager, privilegedSurvDao);
     }
 
+    @Override
```

Comment:
What changed to make these `@Override` annotations required?

Reply:
`@Override` is never required but is considered a good programming practice.  I didn't intentionally make this change, but think that Eclipse did it as part of refactoring when the references were updated.